### ISSUE TITLE
Calculator Haskell file produces incorrect results

### ISSUE THREAD

**Issue by Twigonometry** [Twigonometry/IsabelleDSL#4]
```
*Calculator> eval (St (Int_of_integer 0)) (Add (Int_of_integer 5) (Sub (Int_of_integer 4) (Div (Int_of_integer 4) (GetResult))))
St (Int_of_integer (-4))
*Calculator> eval (St (Int_of_integer 0)) (Add (Int_of_integer 5) (Sub (Int_of_integer 4) (GetResult)))
St (Int_of_integer (-1))
```
However, equivalent Isabelle statements are correct. Exported Python is also correct:
```
=== Results of Exported File ===

0
0.25
```

**Comment by Twigonometry** [comment]
Note: this is separate to the issue with current calculator model not being able to process float values.

**Comment by Twigonometry** [comment]
Issue mentioned above is #19

**Comment by Twigonometry** [comment]
This seems to have fixed itself - strange behaviour I couldn't reproduce after working on other things for a while, perhaps it was an artifact of some caching on Isabelle's part? Either way, closed by [f15a3c1c](https://github.com/Twigonometry/IsabelleDSL/commit/f15a3c1c27ecaeeca4c0f8c9b0c6c9714f8f2312)
